pkg/rescheduler/strategies/duplicates.go

```diff
@@ -21,49 +21,60 @@ import (
 	"strings"
 
 	"k8s.io/kubernetes/pkg/api/v1"
+	//TODO: Change to client-go instead of generated clientset.
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 
 	"github.com/aveshagarwal/rescheduler/pkg/rescheduler/evictions"
 	podutil "github.com/aveshagarwal/rescheduler/pkg/rescheduler/pod"
 )
 
+// This strategy evicts all duplicate pods on node. A pod is said to be a duplicate of other if
+// both of them have same name, kind and are within the same namespace. As of now, this strategy won't
+// evict daemonsets, mirror pods and pods with local storages.
+
 //type creator string
-type DuplicatePodsMap map[string][]*v1.Pod
+type duplicatePodsMap map[string][]*v1.Pod
```

Comment:
I have intentionally kept all exported for time being for faster development, as things will be changing fast.

Reply:
Ok, makes sense. I will change them back.